toolkit/tools/internal/safechroot/safechroot.go

```diff
@@ -342,8 +340,7 @@ func (c *Chroot) MoveOutFile(srcPath string, destPath string) (err error) {
 	srcPathFull := filepath.Join(c.rootDir, srcPath)
 	err = file.Move(srcPathFull, destPath)
 	if err != nil {
-		logger.Log.Errorf("Error moving file '%s'", err)
-		return
+		return fmt.Errorf("Error moving file from %s to %s. Error: %w", srcPath, destPath, err)
```

Comment:
Change to `”failed to move file from (%s) to (%s):\n%w”`. (Similar comment elsewhere.)

Within Go, error messages should start with a lowercase letter. 

You should avoid adding the word “error” within the error message. Instead, you should focus on describing what failed.

Within Go, nested error messages are typically separated by just `:`. Within the Mariner toolkit, we replace the space with a newline, to make the error messages easier to read.

Within the Mariner toolkit, we typically surround values by `()` to make it more obvious when the value is an empty string.

Reply:
Here's a [short explanation](https://staticcheck.dev/docs/checks#ST1005) behind this way of formatting from Go's review comments.

Reply:
And I'm personally fine with either brackets or quotes for strings (if we could unify all logs to one format at some point would be lovely). As long as we're able to make empty strings clearly visible.

Reply:
We should consider Wrapping errors into one another, instead of a string manipulation of the previous error.